projects/design-system-angular/src/lib/accordion/accordion.component.html

```diff
@@ -1,5 +1,6 @@
-<div class="jazz-accordion {{showBorder?'jazz-accordion--bordered':''}}"
-     [attr.aria-multiselectable]="multiSelect ? true : false"
+<div class="jazz-accordion
```

Comment:
Can you try setting the classes by using ngClass? This is typically how dynamic classes are set in angular and it is more explicit for the next developer to understand:
https://www.angularjswiki.com/angular/how-to-add-a-class-based-on-condition-in-angular/
https://www.developer.com/languages/javascript/assigning-dynamic-classes-with-ngclass/

You can delimit the classes with a comma if you need to do multiple classes.